tools/rosmsg/package.xml

```diff
@@ -25,6 +25,8 @@
   <run_depend>rosbag</run_depend>
   <run_depend>roslib</run_depend>
 
+  <test_depend>geometry_msgs</test_depend>
```

Comment:
Currently nothing in this repository depends on `common_msgs`. Therefore I think this should be avoided and local custom messages should be used for these tests.